### DISCUSSION TITLE
informazioni sul software per informativa sito

### DISCUSSION THREAD

**Discussion by mattiastart** [globaleaks/globaleaks-whistleblowing-software#3649] [Q&A]
Dovendo pubblicare un'informativa sul servizio di whistleblowing sul nostro sito internet il nostro ufficio legale ci chiede conferma se il testo riportato di seguito corrisponde con le caratteristiche tecniche del software

**_Modalità di trattamento

I dati saranno trattati con supporti informatici e cartacei che ne garantiscono la sicurezza e la riservatezza. La documentazione in formato cartaceo è limitata al minimo indispensabile e archiviata e custodita in armadi e locali dotati di serrature di sicurezza. In particolare, la trasmissione dei dati forniti dal segnalante mediante compilazione della piattaforma è gestita con protocollo HTTPS. Sono inoltre applicate tecniche di cifratura basate su Algoritmo AES e tutti i dati sono completamente criptati, garantendo in questo modo la riservatezza delle informazioni trasmesse.  Non viene fatto uso di cookies per la trasmissione di informazioni di carattere personale, né vengono utilizzati cookies persistenti per il tracciamento degli utenti. Vengono utilizzati esclusivamente cookies tecnici nella misura strettamente necessaria al corretto ed efficiente utilizzo della piattaforma. L'uso dei cookies di sessione (che non vengono memorizzati in modo persistente sul computer dell'utente e svaniscono con la chiusura del browser) è strettamente limitato alla trasmissione di identificativi di sessione (costituiti da numeri casuali generati dal server) necessari per consentire l'esplorazione sicura ed efficiente della piattaforma._**

Ci sapete indicare se queste informazioni corrispondono con il reale funzionamento del software, soprattutto la parte sui cookies

**Comment by cienne1** [comment]
Avrei necessità della stessa informazione

**Reply by cienne1** [reply]
Sapete darci risposta al quesito?

**Comment by evilaliv3** [comment]
Buonasera, grazie per la domanda.

Si l'informativa riportata è corretta.
In merito ai dati a riposo questi non sono cifrati con AES ma con algoritmi Curve25519, XSalsa20 e Poly1305

L'esatto protocollo crittografico è descritto in: https://docs.globaleaks.org/en/main/security/EncryptionProtocol.html
Voleste includere nell'informativa un link che descrive ogni misucra di sicurezza implementata potreste riferire a: https://docs.globaleaks.org/en/main/security/index.html
